### PR TITLE
feat(extension): update conway era tx details to the latest figma

### PR DESCRIPTION
**Pull request by vetalcore**
# Checklist

- [x] JIRA - [LW-9622](https://input-output.atlassian.net/browse/LW-9622)
- [ ] Proper tests implemented
- [ ] Screenshots added.

---

## Proposed solution

Adjust common tx transformer and related components layout.

## Testing

Visual regression.
Stories are available in the core package under `Sanchonet / ActivityDetail / TransactionDetails`

## Screenshots

Attach screenshots here if implementation involves some UI changes

[LW-9622]: https://input-output.atlassian.net/browse/LW-9622?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/browser-extension-wallet/src/utils/tx-inspection.ts [hunk 1]
```diff
@@ -231,6 +231,22 @@ export const getVoterType = (voterType: Wallet.Cardano.VoterType): VoterTypeEnum
   }
 };
 
+export enum CredentialTypeEnum {
```
Comment (przemyslaw-wlodek) [inline apps/browser-extension-wallet/src/utils/tx-inspection.ts:234]: No need to add `Enum` suffix :)

File: apps/browser-extension-wallet/src/views/browser-view/features/activity/helpers/common-tx-transformer.ts [hunk 2]
```diff
@@ -255,46 +229,73 @@ export const certificateTransformer = (
...
+
+      if ('deposit' in conwayEraCertificate) {
+        const depositPaidInAda = Wallet.util.lovelacesToAdaString(conwayEraCertificate.deposit.toString());
         transformedCertificate.push({
           title: 'depositPaid',
+          info: 'depositPaidInfo',
           details: [
-            `${Wallet.util.lovelacesToAdaString(conwayEraCertificate.deposit.toString())} ${cardanoCoin.symbol}`
+            [
+              `${depositPaidInAda} ${cardanoCoin.symbol}`,
```
Comment (przemyslaw-wlodek) [inline apps/browser-extension-wallet/src/views/browser-view/features/activity/helpers/common-tx-transformer.ts:294]: Like in the previous PR: Please double-check if we don't have a formatter for that 🙏🏻

**Review by przemyslaw-wlodek — APPROVED** [review]
GJ 🍻

**Review by mchappell — CHANGES_REQUESTED** [review]
Blocking merge until comments unresolved in https://github.com/input-output-hk/lace/pull/860 are moved across to this PR.

File: apps/browser-extension-wallet/src/utils/tx-inspection.ts (hunk 1 above)
Comment (vetalcore) [reply apps/browser-extension-wallet/src/utils/tx-inspection.ts:234]: fixed

File: apps/browser-extension-wallet/src/views/browser-view/features/activity/helpers/common-tx-transformer.ts (hunk 2 above)
Comment (vetalcore) [reply apps/browser-extension-wallet/src/views/browser-view/features/activity/helpers/common-tx-transformer.ts:294]: resolved

File: packages/core/src/ui/components/ActivityDetail/components/DetailRowSubitems.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-magic-numbers */
```
[the remark this answers is not in the document]
Comment (vetalcore) [reply packages/core/src/ui/components/ActivityDetail/components/DetailRowSubitems.tsx:1]: resolved

**Comment by vetalcore** [comment]
> Blocking merge until comments unresolved in #860 are moved across to this PR.

@mchappell comments have been resolved/pushed to sanchonet branch

**Review by przemyslaw-wlodek — APPROVED** [review]
Just a CI error to be fixed. ✅

File: apps/browser-extension-wallet/src/stores/slices/activity-detail-slice.ts [hunk 4]
```diff
@@ -222,7 +222,12 @@ const buildGetActivityDetail =
       includedUtcTime: blocks?.utcTime,
       // TODO: store the raw data here and transform it later so we always have the raw data when needed.(LW-9570)
       votingProcedures: votingProceduresTransformer(tx.body.votingProcedures),
-      proposalProcedures: governanceProposalsTransformer(cardanoCoin, tx.body.proposalProcedures),
+      proposalProcedures: governanceProposalsTransformer(
+        cardanoCoin,
+        coinPrices,
+        fiatCurrency,
+        tx.body.proposalProcedures
+      ),
```
Comment (przemyslaw-wlodek) [inline apps/browser-extension-wallet/src/stores/slices/activity-detail-slice.ts:230]: May not be very relevant here, but in general I'd suggest introducing a single `object` param when you have more than ~3 params. This gives you also order flexibility and you don't need to bother with optional properties.

```suggestion
      proposalProcedures: governanceProposalsTransformer({
        cardanoCoin,
        coinPrices,
        fiatCurrency,
        proposalProcedures: tx.body.proposalProcedures
      }),
```

File: apps/browser-extension-wallet/src/views/browser-view/features/activity/helpers/common-tx-transformer.ts [hunk 5]
```diff
@@ -615,11 +650,13 @@ export const governanceProposalsTransformer = (
             },
             {
               title: 'govActionDeposit',
-              details: [governanceActionDeposit?.toString()]
+              // TODO: is it in Ada or lovelaces?
```
Comment (przemyslaw-wlodek) [inline apps/browser-extension-wallet/src/views/browser-view/features/activity/helpers/common-tx-transformer.ts:653]: We need to clarify this before merging + we can use the common formatter if it's lovelaces (AFAIR)

Comment (vetalcore) [reply apps/browser-extension-wallet/src/views/browser-view/features/activity/helpers/common-tx-transformer.ts:658]: @mchappell how do i verify this one?

Comment (vetalcore) [reply apps/browser-extension-wallet/src/views/browser-view/features/activity/helpers/common-tx-transformer.ts:667]: Lovelaces are usually represented as strings, so i'd assume this one is just an ada

**Pull request merged**